app.cfg.example

```diff
@@ -72,6 +72,7 @@ submit_command = /usr/bin/sbatch
 # if value is left/empty everyone can trigger the build
 # value can be a space delimited list of GH accounts
 build_permission = Hafsa-Naeem
+no_build_permission_comment = Label `bot:build` has been set by user `{build_labeler}`, but only users `{build_permission_users}` have permission to trigger the action
```

Comment:
Hmm, not sure if we should let the bot report who they should social engineer into adding labels...

The message should just say

```
Label `bot:build` has been set by user `{build_labeler}`, but this person does not have permission to trigger builds
```

(and similar for deploy, of course)

Reply:
Well, particularly this was suggested/requested by you, no? See https://github.com/EESSI/eessi-bot-software-layer/pull/62#discussion_r1023929322